geometry_msgs/Vector3 pos
string type

float64 min_distance
float64 max_distance

float64 min_value
float64 max_value

float64 field_coeff



